SITCOMTN-023.tex

```diff
@@ -49,13 +49,158 @@
+The fields required for the different tickets include the title, summary, description, component, assignee, priority. 
+The other fields are Start Date (Non milestones), End Date (non milestones), Labels, Reviewers, Approved by.  
+The fields specific to each ticket type is listed below.
+\begin{itemize}
+\item{Milestones}: Milestone level (required), Success Criteria, Due Date, L2 Milestone, L3 Milestones, included Epics
+\item{Epics}: Epic name, Sprint, Story points, Supported Milestones, Included Stories/Bugs
+\item{Story/Bug}: Epic Link, Sprint, Story Points, Supported Epic, Related Issues, Needs to be done before/after
+\item{Tasks}: Story Points, Needs to be done before/after
+\end{itemize}
+
+\subsection{Workflows}
+There will be two different workflows, depending on whether the ticket type is a milestone or not, as shown in Figure \ref{fig:workflow}. 
+The main differences are the acknowledged and invalid states available in the non-milestone case.  
+This comes from the nature of the milestones ticket being only an achieved goal instead of an activity that needs to be done. 
+
+\begin{figure}[h]
+\begin{center}
+\includegraphics[width=0.48\textwidth]{WorkFlowMilestones.png}
+\includegraphics[width=0.48\textwidth]{WorkFlowNonMilestones.png}
+\caption{\label{fig:workflow} Workflows}
+\end{center}
+\end{figure}
+
+
+\subsection{Ticket Creation and Review process}
+To enforce a common process, the workflow has constraints on who is allowed to create tickets and transition them to different state transition. 
+
+Only SIT-COM leads can create, deprecate and/or reopen Milestones tickets. SCLT coordinators and leads can create an Epic. 
+However, only the assignee can invalidate, deprecate, reopen and/or complete the Epic. 
+Stories/bugs are created by a SIT-Com member and the tasks by anyone on the project. 
+
+One particularity of the non-milestone workflow is the required acknowledgement status, only allowed by the assignee, ensuring that the assignee has received the tasks and will act on it. 
+As expected, the assignee as well as the start and end dates are required to transition to the acknowledged state.
+
+Regarding the review process, reviewers must be populated before submitting the ticket for Review as an email will automatically be sent to all reviewers as part of the transition. 
+All reviewers must approve ticket in order for the ticket to transition to Reviewed status, which is done automatically after the approval from the final reviewer. 
+All child and/or included tickets must be approved first. 
+In case of disapproval, the system will automatically put the ticket back into In Progress status and only the assignee can send it back to the In Review state. 
+Each reviewer is required to add a comment as part of the review process, independently of the outcome. 
+The success criteria should be filled in order to guide the reviewers.
+Note that the ticket can not be edited while in In Review, Reviewed state. 
```

Comment:
I think this is unnecessarily rigid... but I voiced that opinion already.
Can tickets at least be commented on AFTER being reviewed?

Reply:
So while I agree we need to change it before I can update the documentation. I will leave it as is and add a ticket for Austin to change it.

Reply:
Might we worth adding the tickets here in the chat before pressing the "resolve" button.